.bazelrc

```diff
@@ -107,6 +107,9 @@ build:untyped-blame --copt=-DTRACK_UNTYPED_BLAME_MODE
 # harden: mark relocation sections read-only
 build:release-linux --linkopt=-Wl,-z,relro,-z,now
 build:release-linux --config=lto-linux --config=release-common
+# Separate config for aarch64, so x86_64 can be differently optimized
+build:release-linux-aarch64 --linkopt=-Wl,-z,relro,-z,now
+build:release-linux-aarch64 --config=lto-linux --config=release-common
```

Comment:
There is perhaps some DRYing up to do here, but let's get the new builds right first and not touch the existing `release-linux` config.

Reply:
I agree, I'm going to take a pass over these options.